src/pages/main/Types.ts

```diff
@@ -8,17 +8,17 @@ export interface NoteSummary {
     workspaceId: string;
     title: string;
     summary: string;
-    author: string | undefined;
-    date: string | undefined;
+    author?: string | undefined;
```

Comment:
what it change?

Reply:
makes author property nullable - I will revert that to be required nonnull